src/transformer/UITransformer.ts

```diff
@@ -520,7 +519,8 @@ export class UITransformer {
         }
 
         // The node must be declared as a constant
-        if (!node.modifiers?.find(m => m.kind == TS.SyntaxKind.ConstKeyword) && !isConstant) {
+        // TODO(pl): according to the compiler, this will always be false, since a VariableDeclaration does not have any modifiers. For Code review, leave it as is.
+        if (!(node as unknown as TS.HasModifiers).modifiers?.find(m => m.kind == TS.SyntaxKind.ConstKeyword) && !isConstant) {
```

Comment:
Note that i don't really understand what is going on here or what the original intent of the code was.

I'm not seeing any specific issues, but i kept the original functionality in.
My recommendation however, is that, after testing, to actually remove this condition